.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Move `.gitignore` outside of `.idea/` directory

Reply:
Created automatically .. deleted

Reply:
You should delete it again, perhaps put `.idea/` directory inside `.gitignore`